### PR TITLE
SHOR-27: Add screens for Full text search and internal pages

### PR DESCRIPTION
**Pull request by swastikpareek**
## PR contains screens for
* Full-text Search Results - Contacts
* Full-text Search Results - Activities
* Full-text Search Results - Contributions
* Full-text Search Results - Event Participants
* Full-text Search Results - Memberships
```shell
$ gulp backstopjs:reference --group search-menu && gulp backstopjs:test --group search-menu
```

### CODE REVIEW

File: scenarios/search-menu.json [hunk 1]
```diff
@@ -38,6 +38,26 @@
       "hideSelectors": ["#mainTabContainer"],
       "onReadyScript": "search/full-text-result.js"
     },
+    {
+      "label": "Full text Search - results - Contact",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Contact&text="
+    },
+    {
+      "label": "Full text Search - results - Activities",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Activity&text="
```
Comment (AkA84) [inline scenarios/search-menu.json]: what is this `cid` param? Is it safe to hardcode its value?

File: scenarios/search-menu.json [hunk 2]
```diff
@@ -38,6 +38,26 @@
...
+    }, 
+    {
+      "label": "Full text Search - results - Contributions",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Contribution&text="
```
Comment (AkA84) [inline scenarios/search-menu.json]: remove this empty `text` param

File: scenarios/search-menu.json [hunk 3]
```diff
@@ -38,6 +38,26 @@
...
+    {
+      "label": "Full text Search - results - Contact",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Contact&text="
+    },
+    {
+      "label": "Full text Search - results - Activities",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Activity&text="
+    }, 
+    {
+      "label": "Full text Search - results - Contributions",
```
Comment (AkA84) [inline scenarios/search-menu.json]: the pattern should be
```
"Full text Search - XYZ - results"
```

File: scenarios/search-menu.json (hunk 1 above)
Comment (swastikpareek) [reply scenarios/search-menu.json]: *csid* is already provided in the URLs for previously written scenarios. And if we don't provide that in the URL the page shows error . Also it gets automatically generated when user clicks on Full text search option from the main navigation.

**Pull request merged**